@stellar/anchor-tests/src/tests/sep31/transactions.ts

```diff
@@ -55,13 +56,15 @@ const canCreateTransaction: Test = {
     hasDirectPaymentServer,
     hasExpectedAssetEnabled,
     differentMemosSameAccount,
+    canCreateQuote,
```

Comment:
The code looks great but there are a few edge cases that make your change tricky and not safe for deployment:
1. There are Anchors that only implement SEP-31 without implementing SEP-38. In those cases, the dependency `canCreateQuote` will fail, as well as this whole sep31 test. You'll need to create a new set of tests that covers anchors implementing both SEP-31+SEP-38. You should create a new file for these tests and decide to add it to the list of tests here:
https://github.com/stellar/stellar-anchor-tests/blob/11bf82c295d01ffea018cf6e3edc048a3156a610/@stellar/anchor-tests/src/helpers/test.ts#L346-L351
2. Additionally, if you want to support anchors with `quotes_required: true`, you should update the `sep31/transactions#canCreateTransaction` test by checking if they are required, and if that's the case, make the same request as the code was doing before and validate if the resulting status code is `400`.

@JakeUrban: do you have any other input here?

Reply:
I've made the following changes:

* Added a set of SEP31+SEP38 tests that execute when `quotes_supported: true`
* Added a check for `quotes_required` in SEP31 tests. If it is false, `canCreateTransaction` and it's dependencies if are executed. If `quotes_required: true`, `canCreateTransaction` expects a 400, and its dependencies don't check for anything.